frontend/src/screens/study-landing.tsx

```diff
@@ -1,137 +1,236 @@
+    const abort = useQueryParam('abort') == 'true';
+    const md = useQueryParam('md') || {};
+    const { demographicSurvey } = useParticipantStudies();
+    const { schedule } = useRewardsSchedule();
+    const nextReward = schedule.find(rewardSegment => !rewardSegment.achieved && rewardSegment.isFuture);
+    const landStudy = useLandStudy();
+
+    // State for controlling the alert visibility
+    const [showAlert, setShowAlert] = useState(false);
+
+    useEffect(() => {
+        const params: LandStudyRequest = {
+            id: Number(studyId),
+            md,
+            consent: consent,
+        };
+        if (abort) {
+            params['aborted'] = LandStudyAbortedEnum.Refusedconsent;
+        }
+
+        landStudy.mutate(params, {
+            onSuccess: setStudy,
+            onError: setError,
+        });
+
+        // Set a timer to show the notification after 1200 milli-seconds
+        const timer = setTimeout(() => {
+            setShowAlert(true);
+        }, 1200);
+
+        return () => {
+            clearTimeout(timer);
+        };
+    }, []);
+
+    // Learners who don't consent won't earn points, so we'll just redirect them home
+    if (!consent) {
+        return <Navigate to='/studies' />;
+    }
 
-    const study = useLoaderData() as ParticipantStudy
-    const learningPathStudies = useLearningPathStudies(study?.learningPath)
+    if (!study) {
+        return <LoadingAnimation message="Loading study" />;
+    }
 
-    if (!study || !study.learningPath) {
-        return <Navigate to='/studies' />
+    if (error) {
+        return <ErrorPage error={error} />;
     }
 
+    const notificationStyles = {
+        position: 'fixed',
+        top: '20px',
+        right: '20px', 
+        zIndex: 9999,
+        borderRadius: '8px', // Rounded corners for a nice look
+        boxShadow: '0px 4px 10px rgba(0, 0, 0, 0.1)', // Shadow for a raised effect
+        padding: '20px', // Add padding for better spacing
+        boxSizing: 'border-box',  
+        fontFamily: 'Arial, sans-serif', // Set custom font family
+        fontSize: '16px', // Set custom font size
+        fontWeight: 'bold', // Set font weight to bold
+        border: '2px solid rgba(60, 179, 113, 0.8)', // Add border with the same color as background
+    };
+
     return (
-        <Page hideFooter
-            data-analytics-view
-            data-analytics-nudge="study-complete"
-            data-content-tags={`,learning-path=${study.learningPath.label},is-new-user=${env.isNewUser},`}
-        >
-            <Card p={{ lg: 'xl' }} m={{ lg: 'xl' }} shadow='md' radius='md'>
-                {study.learningPath?.completed ?
-                    <CompletedLearningPath learningPath={study.learningPath} /> :
-                    <LearningPathProgress learningPath={study.learningPath} studies={learningPathStudies} />
-                }
-            </Card>
-        </Page>
-    )
+        <MantineProvider>
```

Comment:
We already have a <MantineProvider> at the root [here](https://github.com/openstax/kinetic/blob/main/frontend/src/main.tsx#L30) additionally in this file the <Notifications> container is also created. You shouldn't really have to write any new styles or JSX, as we can just use the [notifications system](https://mantine.dev/x/notifications/)  [example here](https://github.com/openstax/kinetic/blob/main/frontend/src/components/notifications/researcher-notification.tsx)

let me know if this makes sense! i think we can probably simplify this PR quite a bit with those changes :)